eng/pipelines/templates/stages/archetype-python-release.yml

```
@@ -94,6 +96,17 @@ stages:
               environment: pypi
```

```suggestion
              environment: package-publish
```

As part of this rework I want to standardize on just the one environment.